src/examples/evas/evas-text.c

```diff
@@ -316,7 +316,11 @@ WinMain(HINSTANCE hInstance EINA_UNUSED
     * example */
    struct text_preset_data init_data =
    {
+#if !defined(_MSC_VER) || defined(HAVE_FONTCONFIG)
       .font = {"DejaVu", "Courier", "Utopia"},
+#else
+      .font = {"C:\\Windows\\Fonts\\Arial.ttf", "C:\\Windows\\Fonts\\Cour.ttf", "C:\\Windows\\Fonts\\Comic.ttf"},
```

Comment:
Comic Sans? Really?

Well, I think we should get similar font alternatives for each one. Courier with Courier New seems fine, but Utopia & Comic veeery very different (one is serif, the other is sans). For replacements I would recommend:

- DejaVu ->Arial (for shape) or Verdana (for spacings)
- Utopia -> Times New Roman (see https://www.fontshop.com/families/utopia/similar).